stage1/init/common/mount.go

```diff
@@ -300,7 +300,15 @@ func ensureDestinationExists(source, destination string) error {
 	return nil
 }
 
-func AppAddMounts(p *stage1commontypes.Pod, ra *schema.RuntimeApp, enterCmd []string) {
+func AppAddMounts(p *stage1commontypes.Pod, ra *schema.RuntimeApp, enterCmd []string) error {
+	sharedVolPath := common.SharedVolumesPath(p.Root)
```

Comment:
this is called in two places, honestly this whole thing should probably be in `common`. maybe as `sharedVolPath, err := common.InitSharedVolumePath()`